### ISSUE TITLE
Netflix 1080p extension?

### ISSUE THREAD

**Issue by informatorius** [Alex313031/quark-player#56]
Would it be possible to add the Netflix 1080p extension? To get Full HD 1920x1080 video stream instead of only 1280x720 max.

https://github.com/jangxx/netflix-1080p

**Comment by MulverineX** [comment]
The cast-labs electron release that Quark Player uses includes VMP and should enable 1080p streaming, are you sure you're not being throttled?

**Comment by informatorius** [comment]
In Quark Player in Netflix when I press Shift+Ctrl+Alt+D during a movie supporting 1080p then it shows "Playing bitrate (a/v): 128 / 662 (1280x720)". So I can only get 720p. The mentioned extension enables 1080p in Chrome Browser. I wish it could also be used in Quark Player

**Comment by MulverineX** [comment]
Looks like these extensions do actually accomplish things, did some testing. This should be implemented

**Comment by informatorius** [comment]
Maybe I see this issue "no 1080p but 720p", because I use the Windows 7 version?

**Comment by MulverineX** [comment]
No, it's because the Netflix web client is shit and likes to reduce bitrate when your Internet connection can handle it
